Tasks/ANTPreview/ant2.js

```diff
@@ -0,0 +1,74 @@
+var tl = require('vso-task-lib');
```

Comment:
not clear on why do we need to clone the js as well if we are not making any changes.
we would have to worry about taking FI from the parent branch of ANT.